packages/bpk-component-slider/src/bpk-slider.scss

```diff
@@ -0,0 +1,89 @@
+@import '~bpk-mixins';
+
+$slider-track-size: 6 * $bpk-one-pixel-rem;
+$slider-control-size: 30 * $bpk-one-pixel-rem;
+$slider-control-inset-size: 20 * $bpk-one-pixel-rem;
+$slider-control-size-large: 42 * $bpk-one-pixel-rem;
+$slider-control-inset-size-large: 30 * $bpk-one-pixel-rem;
+
+.bpk-slider {
+  margin-bottom: $bpk-spacing-xxl;
+
+  &__range {
+    // eslint-disable-next-line selector-attribute-quotes
+    div[class$='1'] {
```

Comment:
Can you explain these styles? Are you not able to apply classes to these elements you are targeting? I there any other way you can achieve the same result?

Reply:
The problem here is that the slider component adds an index to the end of the class which I pass. So the actual class would be `bpk-slider--range + hash + index`
Using css modules I cannot think of a better solution to identify the ranges (the horizontal bars in the slider) individually.